setup.py

```diff
@@ -162,6 +162,8 @@ def write_version(filename=os.path.join(*['airflow',
 qds = ['qds-sdk>=1.9.6']
 cloudant = ['cloudant>=0.5.9,<2.0'] # major update coming soon, clamp to 0.x
 redis = ['redis>=2.10.5']
+kubernetes = ['kubernetes>=3.0.0',
+              'cryptography>=2.0.0']
```

Comment:
Probably you want to update the crypto dependency (see higher up) and reference it from here. I know it could affect non-k8s installs but I don’t think it is smart to maintain two dependencies.